src/renderer/pages/UploaderConfigPage.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div id="config-list-view">
+    <el-row :gutter="20" align="middle" class="config-list">
+      <div class="view-title">
+        {{ $T('SETTINGS') }}
+      </div>
+      <el-col
+        class="config-item-col"
+        v-for="item in curConfigList"
+        :key="item._id"
+        :span="11"
+        :offset="1"
+      >
+        <div
+          :class="`config-item ${defaultConfigId === item._id ? 'selected' : ''}`"
+          @click="() => selectItem(item._id)"
+        >
+          <div class="config-name">{{item.name}}</div>
+          <div class="config-update-time">{{formatTime(item._updatedAt)}}</div>
+          <div class="operation-container">
+            <i class="el-icon-edit" @click="openEditPage(item.type)"></i>
+            <i :class="`el-icon-delete ${curConfigList.length <= 1 ? 'disabled' : ''}`" @click="() => deleteConfig(item._id)"></i>
+          </div>
+        </div>
+      </el-col>
```

Comment:
Should add a area for users to add more configs

Reply:
Somethine like this:

![image](https://user-images.githubusercontent.com/12621342/202598618-d338ecb1-8f28-43e9-83f2-b43c2e50a511.png)

Reply:
This is still under development.